cspell.json

```diff
@@ -1157,6 +1157,7 @@
     "ui",
     "UIColor",
     "UINavigationController",
+    "Uint",
```

Comment:
I don't see `Uint` or `webhook` actually getting used in this PR. Can you revert these additions?

Reply:
The spell check test was failing without this.